aws/rust-runtime/aws-inlineable/src/rds_signer.rs

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//! Code related to creating signed URLs for logging in to RDS.
+//!
+//! For more information, see https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.Connecting.html
```

Comment:
**comment/suggestion**: Since this is inlined for a single service it would be useful to provide an example of how to use this API in conjunction with the actual rds SDK.

Reply:
I couldn't figure out how to get this to work. I'm going to ask the internal team that requested this to write an example and submit a PR.